bitbots_vision/src/bitbots_vision/vision_modules/field_boundary.py

```diff
@@ -557,16 +583,13 @@ def _calculate_field_boundary(_image, _field_color_detector, _x_steps, _y_steps,
 class ReversedFieldBoundaryAlgorithm(FieldBoundaryAlgorithm):
     @staticmethod
     def _calculate_field_boundary(_image, _field_color_detector, _x_steps, _y_steps, _roi_height, _roi_width, _roi_increase, _green_threshold):
-        # type: () -> list
```

Comment:
instead of removing these, you/someone should correct them. In a separate PR.

Reply:
Yea, but now they are definitely incorrect and only on very few functions in the whole vision. We should add them for each function.